azext_iot/central/_help.py

```diff
@@ -1301,7 +1538,45 @@ def _load_central_monitors_help():
         "iot central device twin show"
     ] = """
         type: command
-        short-summary: Get the device twin from IoT Hub.
+        short-summary: Get the device properties from IoT Central.
+    """
+
+    helps[
+        "iot central device twin update"
+    ] = """
+        type: command
+        short-summary: Manage the device properties from IoT Central.
+        examples:
+        - name: Update device properties
+          text: >
+            az iot central device twin update
+            --app-id {appid}
+            --device-id {deviceid}
+            -k {content}
+        - name: Replace device properties
+          text: >
+            az iot central device twin update
+            --app-id {appid}
+            --device-id {deviceid}
+            -k {content}
+            -r
+        - name: Update device component properties
+          text: >
+            az iot central device twin update
+            --app-id {appid}
+            --device-id {deviceid}
+            --co {componentname}
+            -k {content}
+            -r
+        - name: Update device module component properties
```

Comment:
When --replace / -r is used, the action is replace - we should note the difference in the description. Replace vs update may also use different HTTP verbs like put vs patch.

Reply:
Yes, I was using another command: `iot central device twin replace` here for replacing action. Do we want to change it back or only add more explanations in this description?

Reply:
My comment was focused on the help docs / description to clearly illustrate the difference between replace and update operations.

Between `az iot central device twin [replace || update]` you, @yang-hai-feng and @lucadruda can make the call on it. To show past precedence for the iot hub CLI, we consider replace & update of device twin separate commands.

Reply:
Thanks @digimaun, I preferer to use separated commands for `replace` and `update`, @lucadruda, @HangyiWang  any thoughts?

Reply:
Yes. will use `device replace` but not the `device update` with `-r` flag.